src/objectivec/oga_multi_modal_processor.mm

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#import "error_utils.h"
+#import "oga_internal.h"
+#import "ort_genai_objc.h"
+
+@implementation OGAMultiModalProcessor {
+  std::unique_ptr<OgaMultiModalProcessor> _processor;
+}
+
+- (nullable instancetype)initWithModel:(OGAModel*)model error:(NSError**)error {
+  if ((self = [super init]) == nil) {
+    return nil;
+  }
+
+  try {
+    _processor = OgaMultiModalProcessor::Create([model CXXAPIOgaModel]);
+    return self;
+  }
+  OGA_OBJC_API_IMPL_CATCH_RETURNING_NULLABLE(error)
+}
+
+- (nullable OGANamedTensors*)processImages:(NSString*)prompt
+                                    images:(OGAImages*)images
+                                     error:(NSError**)error {
+  try {
+    OGANamedTensors* result = [[OGANamedTensors alloc]
+        initWithCXXPointer:_processor->ProcessImages([prompt UTF8String],
+                                                        [images CXXAPIOgaImages])];
```

Comment:
nit: please run formatter on all objc code

Reply:
Clang-formated, but the result is still pretty much the same with this file. I think it's because of line length limitation.